quinn-proto/src/tests/util.rs

```diff
@@ -375,38 +375,29 @@ impl Write for TestWriter {
     }
 }
 
-pub fn server_config() -> ServerConfig {
-    let key = CERTIFICATE.serialize_private_key_der();
-    let cert = CERTIFICATE.serialize_pem().unwrap();
-
-    let mut crypto = crypto::ServerConfig::new();
-    Arc::make_mut(&mut crypto)
-        .set_single_cert(
-            rustls::internal::pemfile::certs(&mut cert.as_bytes()).unwrap(),
-            rustls::PrivateKey(key.to_vec()),
-        )
-        .unwrap();
-    ServerConfig {
-        crypto,
-        ..Default::default()
-    }
+pub fn server_config(cert_and_key: Option<(Certificate, PrivateKey)>) -> ServerConfig {
```

Comment:
Rather than adding a new `None` argument in a ton of places, should we have two different helper methods?

Reply:
Fair enough, done.